pkg/build/build.go

```diff
@@ -26,6 +29,8 @@ import (
 
 	apko_build "chainguard.dev/apko/pkg/build"
 	apko_types "chainguard.dev/apko/pkg/build/types"
+	apkofs "chainguard.dev/apko/pkg/fs"
+	"github.com/zealic/xignore"
```

Comment:
For future reference, the go-git package is actively maintained and it has support for gitignore patterns too:

https://pkg.go.dev/gopkg.in/src-d/go-git.v4/plumbing/format/gitignore

Reply:
Cool I made an issue for that too.